README.md

```diff
@@ -48,11 +52,11 @@ $client->search()->then(
 );
```

Comment:
While we're at it, does it make sense to update the promise rejection handler to expect an `Exception` here and in the other examples?

Reply:
Thank you for noticing this, with the help of @SimonFrings I searched for the promise rejection handler. I just found this specific example and the demo.php file to adjust the changes.